config/routes.rb

```diff
@@ -188,11 +188,12 @@ def matches?(request)
 
   namespace :further_education_payments do
     namespace :providers do
-      resources :sessions, only: %i[new]
+      resource :sessions, only: %i[new destroy]
```

Comment:
Should we rename this resource to `session` if it's now singular?

Reply:
done